### ISSUE TITLE
cuDNN error:CUDNN_STATUS_EXECUTION_FAILED

### ISSUE THREAD

**Issue by 18842505953** [KaiyangZhou/deep-person-reid#149]
只有在运行hacnn模型时出现这个错误:cuDNN error:CUDNN_STATUS_EXECUTION_FAILED,其他模型都能正常运行，why。我的cmd如下：
python ../deep-person-reid/scripts/main.py --root ../deep-person-reid --app image --loss softmax --label-smooth -s market1501 -a hacnn --optim sgd --lr 0.03 --stepsize[150 , 225] --gamma 0.1 --max-epoch 300 --batch-size 32 --save-dir ../deep-person-reid/TrainAndTest/Pretrain_Parameters_Hacnn_market1501 --gpu-devices 0 --color-jitter --height 160 --width 64

**Comment by KaiyangZhou** [comment]
It seems to be an cudnn error rather than the code. I tried your command on my ubuntu machine with python2 and python3 and both worked.

I googled and found several discussions on this topic [1](https://discuss.pytorch.org/t/cudnn-status-execution-failed/4441/17), [2](https://discuss.pytorch.org/t/cuda-error-runtimeerror-cudnn-status-execution-failed/17625), [3](https://github.com/pytorch/pytorch/issues/5213), though they seem to be irrelevant.

Let me know if you find the solution.

**Comment by 18842505953** [comment]
好的，我先看一下，有结果会给你答复，感谢

**Comment by 18842505953** [comment]
可以提供一下你的电脑的cudnn,cuda,pytorch,python 的版本吗，我觉得是版本匹配的问题
